src/components/Inputs/KeyAndValueList.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div class="relative">
+    <div class="flex gap-4 relative">
+      <div v-if="modelValue && modelValue.length !== 0" class="w-full">
+        <div
+          v-for="(single, index) in modelValue"
+          :key="index"
+          class="relative"
+        >
+          <X
+            class="absolute cursor-pointer right-0 top-[-18px] w-[16px] z-10"
+            @click="removeFields(index)"
+          />
+          <div class="w-full mb-4 grid grid-cols-2 gap-x-4">
+            <div
+              class="relative"
+              :class="{
+                'overflow-hidden': !single.key && hasFocus !== 'key' + index,
+              }"
+            >
+              <FormFieldLabel
+                :is-placeholder="!single.key && hasFocus !== 'key' + index"
+                @click.native="handleLabelClick"
+              >
+                {{ props.text.key_label ? props.text.key_label : 'Key' }} Nr.{{
+                  index + 1
+                }}
+              </FormFieldLabel>
+              <input
+                ref="inputRef"
+                v-model="single.key"
+                class="form-control sm:min-w-[200px] w-full"
+                :class="{ 'sm:min-w-[416px]': long }"
+                :readonly="readonly"
+                type="text"
+                @blur="handleBlur"
+                @focus="handleFocus('key', index)"
+                @input="handleInput('key', index, $event)"
+              />
+            </div>
+            <div
+              class="relative"
+              :class="{
+                'overflow-hidden':
+                  !single.value && hasFocus !== 'value' + index,
+              }"
+            >
+              <FormFieldLabel
+                :is-placeholder="!single.value && hasFocus !== 'value' + index"
+                @click.native="handleLabelClick"
+              >
+                {{ props.text.value_label ? props.text.value_label : 'Value' }}
+                Nr.{{ index + 1 }}
+              </FormFieldLabel>
+              <input
+                ref="inputRef"
+                v-model="single.value"
```

Comment:
forši, ka nav "fact" vairs, bet nu man personīgi skatoties uz kodu "single" baigi neko neizsaka :(  item? singleItem?